internal/requests/structs.go

```diff
@@ -1,5 +1,17 @@
 package requests
 
 type Publisher struct {
-	Name string `json:"name" validate:"required,max=255"`
+	URL         string `json:"url" validate:"required"`
+	Description string `json:"description"`
+	Email       string `json:"email" validate:"email"`
```

Comment:
Doesn't this fail to validate with no email?

Reply:
Nope :-) It ignores it when field is not populated